packages/flutter/test/widgets/nested_scroll_view_test.dart

```diff
@@ -469,12 +468,38 @@ void main() {
         // END
       )),
     );
+
+    PhysicalModelLayer _dfsFindPhysicalLayer(ContainerLayer layer) {
+      expect(layer, isNotNull);
+      Layer child = layer.firstChild;
+      while (child != null) {
+        if (child is PhysicalModelLayer) {
+          return child;
+        }
+        if (child is ContainerLayer) {
+          final PhysicalModelLayer candidate = _dfsFindPhysicalLayer(child);
+          if (candidate != null) {
+            return candidate;
+          }
+        }
+        child = child.nextSibling;
+      }
+      return null;
+    }
+
+    final ContainerLayer nestedScrollViewLayer = find.byType(NestedScrollView).evaluate().first.renderObject.debugLayer;
+    void _checkPhysicalLayer(double elevation) {
+      final PhysicalModelLayer layer = _dfsFindPhysicalLayer(nestedScrollViewLayer);
+      expect(layer, isNotNull);
+      expect(layer.elevation, equals(elevation));
+    }
+
     int expectedBuildCount = 0;
     expectedBuildCount += 1;
     expect(buildCount, expectedBuildCount);
     expect(find.text('Item 2'), findsOneWidget);
     expect(find.text('Item 18'), findsNothing);
-    expect(find.byType(NestedScrollView), isNot(paints..shadow()));
+    _checkPhysicalLayer(0);
```

Comment:
nit: maybe make elevation a required named parameter so it's clear right here what the `0` is?

Reply:
Done